compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolResolver.java

```diff
@@ -172,7 +172,7 @@ public SymbolResolver(CompilerContext context) {
         this.unifier = new Unifier();
     }
 
-    public boolean checkForUniqueSymbol(Location pos, SymbolEnv env, BSymbol symbol) {
+    public boolean checkForUniqueSymbol(Location pos, SymbolEnv env, BSymbol symbol, boolean isEnum) {
```

Comment:
We have to find another approach to deal with enums. Changing public API's just for one type of node is not recommended